### ISSUE TITLE
Suez wont launch on v1.9 after update

### ISSUE THREAD

**Issue by surfac3** [raspiblitz/raspiblitz#3873]
When attempting to launch suez after update the tool is reinstalled 

Once installation is done following is displayed in terminal

<img width="288" alt="Screenshot 2023-05-24 at 13 28 28" src="https://github.com/rootzoll/raspiblitz/assets/89400663/0bcd8d0d-347d-4696-a1ef-8628adf98ab0">

Worth to add i am running reckless 23.05 CLN version.

**Comment by rootzoll** [comment]
CLN 23.05 could be another problem (thats why we still use an older version in v1.9.0), they have some breaking changes. 

For the suez on v1.9.0 please run once: `patch code` on command line (or SSHMENU > UPDATE > PATCH > PATCHMENU > PATCH) & reboot.

**Comment by surfac3** [comment]
Now i am getting different errors 

I fear its connected with the CLN23.05

<img width="961" alt="Screenshot 2023-05-24 at 14 20 15" src="https://github.com/rootzoll/raspiblitz/assets/89400663/ed4d33f3-1de0-4264-94ab-de6d81791b10">

**Comment by openoms** [comment]
> Now i am getting different errors
> 
> I fear its connected with the CLN23.05
> 
> <img alt="Screenshot 2023-05-24 at 14 20 15" width="961" src="https://user-images.githubusercontent.com/89400663/240600579-ed4d33f3-1de0-4264-94ab-de6d81791b10.png">

yes, CLN 23.05 broke a lot of things with the RPC response change.
Will update them as the fixes come.

**Comment by pallablitz** [comment]
> When attempting to launch suez after update the tool is reinstalled
> 
> Once installation is done following is displayed in terminal
> <img alt="Screenshot 2023-05-24 at 13 28 28" width="288" src="https://user-images.githubusercontent.com/89400663/240588787-0bcd8d0d-347d-4696-a1ef-8628adf98ab0.png">
> 
> Worth to add i am running reckless 23.05 CLN version.

Got the same error too on v1.9, but I'm using LND 0.16.2-beta.
